sdk/storage/Azure.Storage.Blobs/tests/BlobClientTests.cs

```diff
@@ -515,6 +515,7 @@ async Task Verify(Task<Response<BlobContentInfo>> upload)
         }
 
         [Test]
+        [LiveOnly]
```

Comment:
Why LiveOnly?  This seems in the range of recordable data.

Reply:
Not a review, but a question.

Reply:
500K is pushing the boundary of what we should be checking in as recordings vs. regenerating on the fly via content hooks -- mostly because we end up re-recording these a lot and I don't want to contribute to longer clone times.  I'll actually just comment out the 500K test for now pending #6781 and remove the LiveOnly.